### ISSUE TITLE
[BUG] Autocomplete does not have an accessible name if you supply an id

### ISSUE THREAD

**Issue by LukeTaylor34** [UKHO/admiralty-design-system#241]
**Describe the bug**
If you add an id attribute then no accessible name is assigned to the autocomplete input.

**To Reproduce**
Steps to reproduce the behaviour:
1. Add a id attribute to the autocomplete component 
2. Inspect the autocomplete input in the DOM

**Expected behaviour**
The input needs to always have an accessible name and we can add an id.

**Screenshots**
![image](https://github.com/user-attachments/assets/e9fd0341-faa9-4315-b305-7dc99f9c8c4c)
![image](https://github.com/user-attachments/assets/09f01119-4ac7-4857-9786-fcd39f21013d)

**Comment by KatiePUX** [comment]
resolved by #253
